<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectivityForm</class>
 <widget class="QWidget" name="ConnectivityForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1130</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>1130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="buttonWifiText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>720</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>41</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(255, 255, 255,0);
color: rgb(208,225,241);</string>
     </property>
     <property name="text">
      <string>Wi-Fi Set-up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBluetoothText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>610</y>
       <width>720</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>41</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(255, 255, 255,0);
color: rgb(208,225,241);</string>
     </property>
     <property name="text">
      <string>Bluetooth Set-up</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInstruction">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>900</y>
       <width>720</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(208,225,241);</string>
     </property>
     <property name="text">
      <string>Connectivity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBack">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1010</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 6px;
background-color: rgb(22,61,86);
color: rgb(208,225,241)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/Images/XatekUI_BackArrow.png</normaloff>:/Images/XatekUI_BackArrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>73</width>
       <height>62</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEnter">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>1010</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>60</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(22,61,86);
color: rgb(208,225,241)</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
     <property name="iconSize">
      <size>
       <width>170</width>
       <height>86</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBluetoothImage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>380</y>
       <width>571</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(255, 255, 255,0);
color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonWifiImage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>571</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(255, 255, 255,0);
color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
